docs/product/components/popovers.html

```diff
@@ -191,9 +204,11 @@
                 data-controller="s-popover"
                 data-s-popover-reference-selector="#reference-element"
                 data-s-popover-placement="auto"
-                data-s-popover-auto-show="true">
+                data-s-popover-auto-show="true"
+                data-s-popover-hide-on-outside-click="if-in-viewport">
```

Comment:
Maybe make this "never" to match the existing behavior?  It seems like "never" is going to be a far more common use-case than "if-in-viewport".

Reply:
Yep, co-signed on this one.

Reply:
Yup, can make this change. I set it to `if-in-viewport` purely to have a place to test it and forgot to change it back.